css/main.css

```diff
@@ -120,57 +120,57 @@ body {
   align-items: center;
   gap: 8px;
 }
-.footer{
+
+.footer {
   background: #fff;
   width: 100%;
   border-top-right-radius: 50px;
   padding: 100px 0;
   min-height: 100vh;
 }
 
-.footer .categories{
+.footer .categories {
   width: 80%;
   margin: 48px auto;
 }
 
-.categories .category{
+.categories .category {
   padding: 12px;
 }
 
-.category .language-skill{
+.category .language-skill {
   display: flex;
   justify-content: space-between;
 }
-.language-skill p{
+
+.language-skill p {
   font-weight: 500;
   font-size: 20px;
-  color: #000000;
+  color: #000;
 }
 
-.category .program-lang{
+.category .program-lang {
   display: flex;
   flex-direction: column;
   align-items: center;
   gap: 12px;
   margin: 24px 0 0;
 }
 
-.program-lang .program{
+.program-lang .program {
   width: 303px;
-    height: 66.72px;
-    background: #f7f7f9;
-    border-radius: 8px;
-    padding: 12px;
-    display: flex;
-    gap: 16px;
-    align-items: center;
+  height: 66.72px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  padding: 12px;
+  display: flex;
+  gap: 16px;
+  align-items: center;
 }
 
-.program p{
+.program p {
   font-weight: 500;
   font-size: 15px;
   line-height: 20px;
   color: #253858;
 }
```

Comment:
For your colors why dont you use [CSS variables](https://www.w3schools.com/css/css3_variables.asp) for them so as for it to be easier to change if you do not like a given color again

Same for `font-family`